package.json

```diff
@@ -64,6 +64,7 @@
     "react-html-parser": "^2.0.2",
     "react-paginate": "^6.5.0",
     "shortid": "^2.2.15",
+    "storybook": "^6.0.28",
```

Comment:
should this be under `devDependencies`?

Reply:
Good q, though I think we deploy to ghp.wellcomecollection.org/corporate-components/ - so does that mean it should stay in depdendencies?

Reply:
...then again, that URL is not really production, is it? It's just a deployed version of a testing environment. I think I'll move storybook to devDependencies.

Reply:
Absolutely. The storybook dependency would become part of the npm package which we definitely don't want